DocumentDB/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  common_tags = merge(
+    var.default_tags,
+    {
+      CostCentre = var.billing_code
+    }
+  )
+}
```

Comment:
Nice idea here to let people add more tags to the `common_tags`.  I'll definitely add this into future modules I build.

Reply:
Thank you! It was a nice discovery for sure. 😊